ssr-express/demo/runtimeVersion.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const express = require('express');
+const path = require('path');
+const app = express();
+const AmpSsrMiddleware = require('../index.js');
+const runtimeVersion = require('amp-toolbox-runtime-version');
+
+// Wrap the current version in our own function. The function has no parameters and returns
+// a Promise<Number>.
+// This function will be called every time a new AMP is transformed.
+const currentVersion = () => runtimeVersion.currentVersion();
```

Comment:
this is not needed, simply pass `runtimeVersion.currentVersion` as an option

Reply:
Tried that, but it breaks with the following error:

```
TypeError: Cannot read property 'getAndUpdateRelease_' of undefined
```